### PR TITLE
Feat: Add icons for neuron settings

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

New icons and one changed icon that will be used in the new neuron settings.

# Changes

* New Icons: `IconStakedMaturity`, `IconClockDissolving`, `IconClockNoFill`, `IconLockClosed`.
* Changed icon: `IconLockOpen`

# Screenshots

Screenshot 2023-07-17 at 09 08 40
Screenshot 2023-07-17 at 09 09 18

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker please review

**Comment by lmuntaner** [comment]
@mstrasinskis please review

**Comment by peterpeterparker** [comment]
Do you have an idea of generic names for `IconClockDissolving` and `IconStakedMaturity` ?

**Comment by lmuntaner** [comment]
> Do you have an idea of generic names for `IconClockDissolving` and `IconStakedMaturity` ?

To be honest now. I would have preferred something generic. But they are pretty weird icons 😂 . It's hard to understand what they represent. Is it a broken clock? Is it a sunrise? Those names might create even more confusion.

I'm happy to change if you have some suggestions.

**Comment by peterpeterparker** [comment]
`IconClockDissolving` -> `Pace` (source: https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Apace%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048=

`IconStakedMaturity` seems quite particular indeed

**Comment by lmuntaner** [comment]
@peterpeterparker renamed and I also added the light theme screenshot. I realized there was an issue with it.

File: src/lib/icons/IconLockOpen.svelte [hunk 1]
```diff
@@ -1,15 +1,25 @@
 <!-- source: https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Alock_open%3A -->
```
Comment (peterpeterparker) [inline src/lib/icons/IconLockOpen.svelte:1]: Isn't it now a custom icon?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

Just added a minor comment if you can have a look before merge.

File: src/lib/icons/IconLockOpen.svelte (hunk 1 above)
Comment (lmuntaner) [reply src/lib/icons/IconLockOpen.svelte]: yes, thanks for noticing!

**Pull request merged**